index.js

```diff
@@ -34,7 +34,7 @@ exports.connect = function (url, name, callback) {
 
 exports.close = function (url) {
 	if(!conns[url]) return;
-	conns[url].end();
+	conns[url].socket.destroy();
```

Comment:
i think you want to change this to 

```javascript 
cons[url].end();
```

My understanding is that socket.destroy() is only necessary in case of errors.